modules/mod_tags_similar/src/Helper/TagsSimilarHelper.php

```diff
@@ -194,7 +194,13 @@ public static function getList(&$params)
 			$query->order($query->rand());
 		}
 
-		$query->setLimit((int) $params->get('maximum', 5));
+		$maximum = (int) $params->get('maximum', 5)
```

Comment:
```suggestion
		$maximum = (int) $params->get('maximum', 5);
```

This should fix the failing tests